src/components/CharacterOrderDialog.vue

```diff
@@ -280,9 +280,10 @@ export default defineComponent({
           selectedCharacter.value = sampleCharacterOrder.value[0];
 
           // 保存済みのキャラクターリストを取得
-          characterOrder.value = store.state.userCharacterOrder.map(
-            (speakerUuid) => characterInfosMap.value[speakerUuid]
-          );
+          // FIXME: 並び替えると不明なキャラがリセットされる。
```

Comment:
closeDoalogが呼ばれるときにリセットされるので、並び替えしなくてもリセットされると思います。
なのでこんな感じ…？

```suggestion
          // FIXME: 不明なキャラを無視しているので、保存時にリセットされる。
```